extensions/crossmodel-lang/package.json

```diff
@@ -94,27 +97,29 @@
     "@eclipse-glsp/layout-elk": "2.0.0",
     "@eclipse-glsp/server": "2.0.0",
     "chalk": "~4.1.2",
-    "chevrotain": "~10.4.2",
+    "chevrotain": "~11.0.3",
```

Comment:
@martin-fleck-at 

When @ozcanxbreeze worked on the Langium implementation he ran into some issue related to this PR in chevrotain: https://github.com/Chevrotain/chevrotain/pull/1917.

Because of this reason we didn't use a higher chevrotain version. Is this problem not occuring with the latest chevrotain (since I see you upgraded it?

Reply:
Yes that was fixed in https://github.com/Chevrotain/chevrotain/commit/d50420dc64e52310250136dc9e848057f2e8afb7, so very recently!